File: movie-infrastructures/storage/src/main/java/com/movie/storage/movie/repository/SeatJpaRepository.java
package com.movie.storage.movie.repository;

import com.movie.storage.movie.entity.SeatEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface SeatJpaRepository extends JpaRepository<SeatEntity, Long> {

    List<SeatEntity> findByIdIn(List<Long> ids);

}

File: movie/infra/src/main/kotlin/com/example/movie/persistence/seat/repository/SeatJpaRepository.kt
package com.example.movie.persistence.seat.repository

import com.example.movie.persistence.seat.entity.SeatEntity
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface SeatJpaRepository : JpaRepository<SeatEntity, Long> {
    fun findAllByIdIn(seatIds: Collection<Long>): List<SeatEntity>
}